src/core/getFrameMetadata.ts

```diff
@@ -29,14 +19,18 @@ export const getFrameMetadata = function ({
   const metadata: Record<string, string> = {
     'fc:frame': 'vNext',
   };
-  buttons.forEach((button, index) => {
-    metadata[`fc:frame:button:${index + 1}`] = button.label;
-    if (button.action) {
-      metadata[`fc:frame:button:${index + 1}:action`] = button.action;
-    }
-  });
   metadata['fc:frame:image'] = image;
-  metadata['fc:frame:post_url'] = post_url;
+  if (buttons) {
```

Comment:
Why changing this code? Types already checked for buttons to exist.

Reply:
I updated `FrameMetadata` fields. `buttons` and `post_url` are now considered optional (as they are in the Farcaster Frames API).

Reply:
Wait, Buttons are optional, that seems very odd!

Reply:
Agreed. Frames aren't much different than posting an image preview of a site without the buttons... but that's what their docs say:

<img width="838" alt="Screenshot 2024-01-31 at 1 42 07 PM" src="https://github.com/coinbase/onchainkit/assets/3264051/528a9f48-f3dc-4327-89b4-496db00afef7">

Reply:
sad, but I can see their point

Reply:
Thank you for triple checking their standard